### PR TITLE
Add link to GI Bill statement

### PR DESCRIPTION
**Pull request by peggygannon**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3482

### CODE REVIEW

**Comment by peggygannon** [comment]
@rroueche @bethpotts Here is PR for adding a link to the Post-9/11 GI Bill Benefits statement.

**Comment by bethpotts** [comment]
@peggygannon I like the idea of a button on that GI Bill page. Let's talk offline about whether it would be a green button or a blue one.
@rroueche Can we get the login info we need to see the tool?:
screen shot 2017-07-19 at 12 23 50 pm

**Comment by rroueche** [comment]
@peggygannon and @bethpotts 

to see it in staging: https://staging.vets.gov/education/gi-bill/post-9-11/ch-33-benefit
Authenticate: veterans, am3rica
Sign In: vets.gov.user+228@gmail.com, pw: Zm9ydHkgdHdv

**Comment by bethpotts** [comment]
@rroueche Thanks! Very helpful to be able to see the page we're linking to. It's not a tool, really, is it? It seems informational, no action to take. Am I right?

**Comment by rroueche** [comment]
You can print your Statement of Benefits, but that's the only interactive element

**Comment by peggygannon** [comment]
@rroueche @bethpotts An update on this: 

I made the link a blue button on this page: https://www.vets.gov/education/gi-bill/, but the Content team thinks it best to keep it as a link. So you can disregard my 2nd bullet point in my earlier note! 

The PR is ready for your review.

**Comment by peggygannon** [comment]
@rroueche @raquelromano-gov Here is PR for adding navigation to the Post-9/11 GI Bill benefits to the static pages.

File: content/pages/education/eligibility.md
Comment (raquelromano-gov) [inline content/pages/education/eligibility.md]: Missing a leading /
(/education/gi-bill/post-9-11/ch-33-benefit)

File: content/pages/education/gi-bill/post-9-11.md
Comment (raquelromano-gov) [inline content/pages/education/gi-bill/post-9-11.md]: Same here.

**Review by raquelromano-gov — COMMENTED** [review]
Will leave content review to @rroueche 

I also linked this to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3482 so we can track it in waffle

**Comment by bethpotts** [comment]
@rroueche What's the launch target for this? Can you hold off your review of our static content changes until Monday mid-day ET while we finalize the content for those GI Bill page intro paragraphs? We got sidetracked with a couple other things today!
cc @peggygannon

**Comment by rroueche** [comment]
@peggygannon 'benefits summary' is a term for a different official letter from the VA (a doc unrelated to ed benefits) so I updated to the term we use on the page - statement of benefits. I know in another issue you recommended using "benefits statement" instead of "statement of benefits" but I want to stick in the paradigm of "certificate of ____" "statement of ____" so that the print-out does match with VA's provision of official docs. Since that's the doc that students will be using to prove their Ch33 benefit status to their schools.

I'm running these changes by JR today and we'll see how soon we can launch thereafter!

**Review by bethpotts — APPROVED** [review]
screen shot 2017-07-25 at 8 12 20 pm
My only question to raise is about the breadcrumb on the Statement of Benefits page itself. Not sure where to flag this, so putting it here. It says "Check Benefit"--shouldn't that be plural?: Check Benefits

Content on the three pages in this PR (Eligibility, GI Bill, Post-9/11 GI Bill) looks good!

File: content/pages/education/gi-bill/post-9-11.md
Comment (raquelromano-gov) [inline content/pages/education/gi-bill/post-9-11.md]: One last missing / in this link

**Comment by leannammiller** [comment]
@rroueche what's the status here?

**Comment by rroueche** [comment]
Got approval from JR, doing a final check now and if all looks good I'll merge!

@bethpotts thanks for your comment about Check Benefit, I'll add another ticket to think about whether we need/want to change that... I believe Vets.gov needs to do a URL/page title standardization audit in general (is that on the Nebula board still?) So perhaps this could be added as a bullet point example in that issue too!

**Review by rroueche — APPROVED** [review]

**Pull request merged**
